media/css/firefox/whatsnew/includes/_footer.scss

```diff
@@ -0,0 +1,64 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// * -------------------------------------------------------------------------- */
+// Footer overrides
```

Comment:
I kind of like using the white footer as a default WNP footer because it's really clean and a bit of a standout from the regular Firefox pages (with grey footer)

But I'm not sure how often it's actually used or if it should be an optional import (not included in base styles, allow inheritance of grey background firefox footer)

Reply:
Makes sesne to me, as the white footer styles have been the default on most WNPs for some time. We can always change it to be optional in the future if needed.